test/api.js

```diff
@@ -4,9 +4,10 @@
 
 const url = require('url');
 const assert = require('insist');
-const bidcrypto = require('browserid-crypto');
 const nock = require('nock');
 const buf = require('buf').hex;
+const generateRSAKeypair = require('generate-rsa-keypair');
```

Comment:
> i don't anticipate us generating prod keys with it

Agreed, but these things can easily propagate to unintended uses once people notice them